Assets/MixedRealityToolkit-Examples/Sharing/Scripts/ImportExportAnchorManager.cs

```diff
@@ -1,13 +1,16 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
+using HoloToolkit.Sharing;
```

Comment:
Is this HoloToolkit.Sharing using for the native code base?

Reply:
Yes, these will be updated when the base MRTK DLL's / autogenerated files are updated.

Feature Branch isn't ready to be merged, but it's almost there.